package.json

```diff
@@ -71,7 +71,7 @@
     "postinstall-postinstall": "^2.1.0",
     "prettier": "^2.0.5",
     "react": "16.11.0",
-    "react-native": "0.62.2",
+    "react-native": "0.63.2",
```

Comment:
Are we bumping the version?

Reply:
Yes, this version upgrade was due since we added Pressable component which is released in RN 0.63. Upgraded it in this PR to fix the failing of testcases.